SYS_ATL/LoopIR_compiler.py

```diff
@@ -676,6 +676,7 @@ def comp_s(self, s):
                         assert isinstance(s.args[i], LoopIR.WindowExpr)
                         data, _ = self.window_struct_fields(s.args[i])
                         d[f'{arg_name}_data'] = data
+                        d[f'{arg_name}_int'] = self.env[s.args[i].name]
```

Comment:
Note that we needed this because AMX tile require `const`

Reply:
@yamaguchi1024 - This is LoopIR_compiler.py

Reply:
You can't pass `const` between C++ and C. It's an ABI incompatibility.

Reply:
There is no external linkage here or anything, so I'm not sure what is the problem?

Reply:
There _has to be_ external linkage here, because we must not compile our generated code as C++. The headers have (or ought to have)
```
#ifdef __cplusplus
extern "C" {
#endif 

...

#ifdef __cplusplus
}
#endif
```
in them.

Reply:
Ok. If we use C `const` and keep this code, would that be an issue?

Reply:
Depends where the const is... an extern-C const function parameter has the same ABI as a non-const parameter in both languages. However, changing a _struct field_ to `const` has a different ABI.